skbio/stats/power.py

```diff
@@ -173,28 +177,34 @@ def subsample_power(test, samples, draw_mode='ind', alpha_pwr=0.05,
         "ind" mode should be used.
     alpha_pwr : float, optional
         The critical value used to calculate the power.
-    min_observations : unsigned int, optional
```

Comment:
Please note the removal of this parameter (including the reason why it's removed) under "Backward-incompatible changes".